graphql.py

```diff
@@ -133,21 +134,21 @@ def fetch_user_list(username: str):
     return json.loads(response.content, object_hook=to_object)
 
 
-def update_series(media_id: int, progress: int, status: str):
+def update_series(media_id: int, progress: int, status: str, scoreRaw: int):
     if ANILIST_SKIP_UPDATE:
         logger.warning("[ANILIST] Skip update is enabled in settings so not updating this item")
         return
     query = """
-        mutation ($mediaId: Int, $status: MediaListStatus, $progress: Int) {
-            SaveMediaListEntry (mediaId: $mediaId, status: $status, progress: $progress) {
+        mutation ($mediaId: Int, $status: MediaListStatus, $progress: Int, $scoreRaw: Int) {
+            SaveMediaListEntry (mediaId: $mediaId, status: $status, progress: $progress, scoreRaw: $scoreRaw) {
                 id
                 status,
                 progress
```

Comment:
```suggestion
                progress,
                scoreRaw
```
Otherwise, you're not setting the score. But also, scoreRaw should only be part of the update if it's `!= 0`.

Reply:
I put a check for scoreRaw and now it omits the variable from the update. Seems to be working. Adding the scoreRaw to that part of the request gives me a 400, don't know why but updating the score seems to work without it (still trying to figure out graphql)

Reply:
I'm in the progress of migrating the graphql queries to https://github.com/profusion/sgqlc, which allows dynamic queries.

Once I'm done with that we can continue adding the score.

The current problem is that you have to change multiple lines in the graphql query in order to set or omit the score field.